helm/alfresco-infrastructure/Chart.yaml

```diff
@@ -1,4 +1,5 @@
 apiVersion: v1
 description: A Helm chart Providing the required Alfresco infrastructure. This includes nginx-ingress, activemq, alfresco-api-gateway, alfresco-identity-service, and the content services volume.
```

Comment:
Shouldn't we change this? We are not including Ingress, ActiveMQ and the others in the stable version, right?